apps/expo-go/src/components/NavigationScrollView.tsx

```diff
@@ -18,27 +14,12 @@ type StyledScrollViewProps = PropsWithChildren<
     }
 >;
 
-function useThemeBackgroundColor(props: StyledScrollViewProps, colorName: ThemedColors) {
-  const theme = useTheme();
-  const themeName = theme.dark ? ColorTheme.DARK : ColorTheme.LIGHT;
-  const colorFromProps =
-    themeName === ColorTheme.DARK ? props.darkBackgroundColor : props.lightBackgroundColor;
-
-  if (colorFromProps) {
-    return colorFromProps;
-  } else {
-    return Colors[themeName][colorName];
-  }
-}
-
-export default function NavigationScrollView(props: StyledScrollViewProps) {
+export default function NavigationScrollView({ style, ...otherProps }: StyledScrollViewProps) {
   const ref = useRef(null);
-  const { style, ...otherProps } = props;
-  const backgroundColor = useThemeBackgroundColor(props, 'bodyBackground');
 
   useScrollToTop(ref);
 
   const ScrollView = Platform.OS === 'android' ? RNScrollView : RNGHScrollView;
 
-  return <ScrollView style={[{ backgroundColor }, style]} {...otherProps} ref={ref} />;
+  return <ScrollView style={style} {...otherProps} ref={ref} />;
```

Comment:
Were these background color changes tested on Android?

Reply:
Just got a test device, and the background color seems to be correct. However, I have pushed the small tweak for the Account "modal".